src/test/java/pro/trevor/tankgame/util/TestUtilities.java

```diff
@@ -14,7 +14,9 @@ public static State generateBoard(int width, int height, IUnit... units) {
         for (IUnit unit : units) {
             board.putUnit(unit);
         }
-        return new State(board, buildTestCouncil(0, 0, 0), new AttributeList<>());
+        State state = new State(board, buildTestCouncil(0, 0, 0), new AttributeList<>());
+        state.put(Attribute.RANDOM, new Random(0));
```

Comment:
I don't think we should put a random in the default test state.  Unit tests should be fairly intentional and I think if someone want to test code using randomness they should define it specifically for there test.